generator/README.md

```diff
@@ -82,14 +82,15 @@ modules:
                             # Required if context is configured on the device.
 
     lookups:  # Optional list of lookups to perform.
-              # This must only be used when the new index is unique.
+              # If keep_old is the default of false, this must only be used when the new index is unique.
```

Comment:
Should `keep_old` be `keep_sources_indexes`?

Reply:
Done